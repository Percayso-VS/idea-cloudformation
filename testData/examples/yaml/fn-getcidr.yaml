AWSTemplateFormatVersion: '2010-09-09'
Resources:
  myEC2Instance:
    Type: "AWS::EC2::Instance"
    Properties:
      ImageId: !GetCidr [ !GetCidr [ "a", 100 ], 100, 22 ]
      InstanceType: !GetCidr [ "a" ]
