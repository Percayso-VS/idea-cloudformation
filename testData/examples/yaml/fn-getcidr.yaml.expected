AWSTemplateFormatVersion: '2010-09-09'
Resources:
  myEC2Instance:
    Type: "AWS::EC2::Instance"
    Properties:
      ImageId: !GetCidr [ !GetCidr [ "a", 100 ], 100, 22 ]
      InstanceType: 0@<!GetCidr [ "a" ]>

0: GetCidr expects two or three arguments
